src/wp-includes/class-wp-user.php

```diff
@@ -62,7 +65,7 @@ class WP_User {
 	 * @var bool[] Array of key/value pairs where keys represent a capability name
 	 *             and boolean values represent whether the user has that capability.
```

Comment:
See above. Not a problem in this PR, but since we're touching it, it would be good to fix this incorrect type annotation.
```suggestion
	 * @var array<string, bool> Array of key/value pairs where keys represent a capability name
	 *                          and boolean values represent whether the user has that capability.
```